core/services/job/models.go

```diff
@@ -273,6 +273,18 @@ func (r *JSONConfig) Scan(value interface{}) error {
 	return json.Unmarshal(b, &r)
 }
 
+func (r JSONConfig) EVMChainID() (int64, error) {
+	i, ok := r["chainID"]
+	if !ok {
+		return -1, fmt.Errorf("%w: chainID must be provided in relay config", ErrNoChainFromSpec)
+	}
+	f, ok := i.(float64)
+	if !ok {
+		return -1, fmt.Errorf("expcted float64 chain id but got: %T", i)
+	}
+	return int64(f), nil
```

Comment:
just curious why we store it as float64 if we always convert to int64 anyway?

Reply:
Because it is being deserialized upstream by a default JSON decoder, which defaults to float64 for all numbers.
Ideally we could tighten that up too, but seemed out of scope.

Reply:
oof this is bad, we should use json.Number. Is there a ticket to fix this?